bw2io/strategies/ecospold2.py

```diff
@@ -262,7 +847,52 @@ def fix_ecoinvent_flows_pre35(db):
 
 
 def drop_temporary_outdated_biosphere_flows(db):
-    """Drop biosphere exchanges which aren't used and are outdated"""
```

Comment:
I think this should be reflected in the function description, not just in the 'Returns' section of the docstring.

Reply:
added